saleor/payment/migrations/0008_merge_20190214_0447.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.1.4 on 2019-02-14 10:47
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('payment', '0007_auto_20190206_0938'),
+        ('payment', '0007_auto_20190125_0242'),
+    ]
+
+    operations = [
```

Comment:
Empty migration?

Reply:
It is because after this PR is submitted, there is migration with same number ```007``` is made in saleor. So I have to merge these two migrations as django reminds me.